cypress/integration/transfer/transferring-items.feature

```diff
@@ -0,0 +1,30 @@
+Feature: Transferring items between lists
```

Comment:
What happens when items are doubled clicked while another one or multiple are selected?

Reply:
The item is transferred as normal. I didn't create a scenario for this because I thought it was covered in the other scenario by *not being mentioned*. Do you think it should be have its own scenario?

Reply:
I def. think we should add any behavior. Anything that's not documented will be either unclear (and therefore the dev will come up with his/her own solution) or it'll be simply buggy until a user notices it and reports a bug

Reply:
Agreed, I've added a scenario for this:
```
  Scenario: Double click an item when others are highlighted
    Given the options or selected list has two or more items
    And one or more items are highlighted
    When an item is double clicked
    Then the item is removed from its current list
    And the item is visible in the other list
    And the previously highlighted items are no longer highlighted
```